src/test/javascript/spec/component/programming-exercise/programming-exercise-update.component.spec.ts

```diff
@@ -73,48 +63,39 @@ describe('ProgrammingExercise Management Update Component', () => {
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            imports: [ArtemisTestModule, MockModule(NgxDatatableModule), MockModule(NgbModule), MockModule(OwlDateTimeModule)],
+            imports: [ArtemisTestModule, MockModule(NgxDatatableModule), MockModule(NgbAlertModule)],
             declarations: [
                 ProgrammingExerciseUpdateComponent,
-                MockComponent(AlertComponent),
-                MockComponent(AlertErrorComponent),
-                MockComponent(HelpIconComponent),
+                NgForm,
```

Comment:
I think you should also be able to mock this, otherwise the changes look good to me 👍🏼 
```suggestion
                MockDirective(NgForm),
```

Reply:
Yeah, I also thought so. Actually I should be able to replace all not mocked directives by importing `MockModule(FormsModule)`, but for some reason this fails some of the tests 😞

Reply:
Oh, I see. Then it's fine like this, I guess 😅